apps/fluent-tester/src/FluentTester/theme/ThemePickers.tsx

```diff
@@ -1,26 +1,38 @@
 import * as React from 'react';
-import { View, Picker, StyleSheet } from 'react-native';
+import { View, StyleSheet, Platform } from 'react-native';
 import { Text } from '@fluentui-react-native/experimental-text';
 import { lightnessOptions, testerTheme } from './CustomThemes';
 import { themeChoices, ThemeNames } from './applyTheme';
 import { brandOptions, OfficeBrand } from './applyBrand';
+import { Theme, useTheme } from '@fluentui-react-native/framework';
+import { themedStyleSheet } from '@fluentui-react-native/themed-stylesheet';
+
+const isAndroid = Platform.OS == 'android';
+
+// To be replaced with single import when Windows support comes in.
+const Picker = isAndroid ? require('@react-native-picker/picker').Picker : require('react-native').Picker;
```

Comment:
react-native-picker already supports windows (but not win32). I'm guessing you mean once someone updates the windows test app to support it? 
Also I feel like using `require()` inside typescript is bad practice although I'm not actually sure..

Reply:
Yep, windows test app needs to be supported.
Regarding `require()` I found that it was the best way to use shared logic around a component coming from different packages.